src/BasePresenter.php

```diff
@@ -71,6 +71,22 @@ public function getRouteKeyName()
     }
 
     /**
+     * Retrieve model for route model binding.
+     *
+     * @param mixed $routeKey
+     *
+     * @return mixed
+     */
+    public function resolveRouteBinding($routeKey)
+    {
+        if (method_exists($this->wrappedObject, 'resolveRouteBinding')) {
```

Comment:
I didn't know about that, now I learned something new :-)

Shouldn't the same thing be changed in `__call()` too then?